src/Main.java

```diff
@@ -2,5 +2,91 @@
 public class Main {
     public static void main(String[] args) {
 
+        System.out.println("Задание 1");
+
+        int clientOS = 0;
+
+        if (clientOS == 0) {
+            System.out.println("Установите версию приложения для iOS по ссылке");
+        } else if (clientOS == 1) {
+            System.out.println("Установите версию приложения для Android по ссылке");
+        }
+
+        System.out.println("Задание 2");
+
+        int clientOS2 = 0;
```

Comment:
переменные с цифрами на концах - это всегда плохо. Сигнал, что с переменными идёт неправильная работа. Можно просто переиспользовать уже созданную переменную вместо создания новой с непонятным названием
```suggestion
        clientOS = 0;
```